students/M1lya1/hw-3/task1.md

```diff
@@ -0,0 +1,23 @@
+1. 1   a обьявлена до вызова функции
```

Comment:
Not exactly... 
because of the closure in JS, the context is created when a you declare a functions, not when you run it.

Reply:
1. undefined